from config.exception.Exception import BaseCustomException

class CustomException:
    pass